pkg/operator/ceph/object/bucket/provisioner_test.go

```diff
@@ -318,7 +319,7 @@ func TestProvisioner_setAdditionalSettings(t *testing.T) {
 	t.Run("quotas are enabled and need updated enabled", func(t *testing.T) {
 		putValsSeen := []string{}
 		p := newProvisioner(t,
-			`{"user_quota":{"enabled":true,"max_size":1,"max_objects":1}}`,
+			`{"enabled":false,"check_on_raw":false,"max_size":1,"max_size_kb":0,"max_objects":1}`,
```

Comment:
The name of the test and the content of the provisioner don't seem to match. I think this should be `"enabled":true` as was the original definition.

Reply:
That is definitely a typo.